Notifications/Notifications/Presentation/Settings/NotificationsSettingsViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NotificationsSettingsViewModel.swift
+//  Notifications
+//
+//  Created by Saeed Bashir on 12/13/24.
+//
+
+import Foundation
+import Core
+import SwiftUI
+
+public class NotificationsSettingsViewModel: ObservableObject {
+    @Published var showError: Bool = false
+    public var hasPermission: Bool = false
```

Comment:
Since we will have a list of push settings, perhaps we need to develop a way to interact with the list of ‘settings’.
Same in View - we need to have a ForEach that will show a list of all settings (I realise we have an API now, but we need to interact with many settings, and for the API we could just have an empty function at the moment). Just add one/two more settings to the ‘Discussion Activities'.

Reply:
We do have plan for adding more option but are not sure when they are coming, thats why I keep it simple according to the requirments.

Reply:
Yes, in the future, we will be getting more push notification settings, but when we are not sure, that's why I keep it simple as per the current requirements. When they come, we will plan it accordingly.

Reply:
got it